packages/ant.rb

```diff
@@ -3,25 +3,12 @@
 class Ant < Package
   description 'Apache Ant is a Java library and command-line tool whose mission is to drive processes described in build files as targets and extension points dependent upon each other.'
   homepage 'http://ant.apache.org/'
-  version '1.10.7'
+  version '1.10.7-1'
```

Comment:
New version available?  See https://downloads.apache.org/ant/source/apache-ant-1.10.9-src.tar.xz.

Reply:
updated

Reply:
If we have to rebuild binaries, it only makes sense to perform the upgrade also.